sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -64,6 +71,10 @@ public ActivityLifecycleIntegration(
     if (buildInfoProvider.getSdkInfoVersion() >= Build.VERSION_CODES.Q) {
       isAllActivityCallbacksAvailable = true;
     }
+
+    // we only track app start for processes that will show an Activity (full launch).
+    // Here we check the process importance which will tell us that.
+    foregroundImportance = isForegroundImportance(this.application);
```

Comment:
hm, just thinking if this will be called in case of the warm app start? This integration is added in `Application.onCreate`, which won't be called in case of warm start, right?

Reply:
it's in the ctor of the integration, so it'll be called too.

Reply:
But the integration ctor itself - will it be called in case of the warm start, if the process wasn't killed?